Source/ablastr/fields/PoissonSolver.H

```diff
@@ -109,6 +113,8 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
 {
     using namespace amrex::literals;
 
+    WARPX_PROFILE("computePhi");
```

Comment:
Inside ABLASTR we use:
```suggestion
    BL_PROFILE("computePhi");
```
or
```suggestion
    ABLASTR_PROFILE("computePhi", false);
```
to avoid access to the `WarpX::` global variable space - which for instance ImpactX has no access to.